packages/react/src/textarea/Textarea.tsx

```diff
@@ -0,0 +1,77 @@
+import { type ComponentPropsWithRef, type ReactNode, forwardRef } from "react";
+import TextareaAutosize from "react-textarea-autosize";
+
+import type { ExtendProps } from "../utils";
+
+import { Box } from "../box";
+import { Flex } from "../flex";
+import { extractSprinkles } from "../sprinkles";
+import * as styles from "./Textarea.css";
+
+type TextareaProps = ExtendProps<
+  ComponentPropsWithRef<"textarea">,
+  ComponentPropsWithRef<typeof Box>,
+  {
+    autoResize?: boolean;
+    bottomSection?: ReactNode;
+    disabled?: boolean;
+    error?: boolean;
+    readonly?: boolean;
```

Comment:
Text area already has read only props. We don't need it explicitly.

Reply:
addressed